src/dispatchers/mouseDispatcher.ts

```diff
@@ -130,7 +130,7 @@ export module Dispatcher {
       var newMousePosition = this._computeMousePosition(e.clientX, e.clientY);
       if (newMousePosition != null) {
         this._lastMousePosition = newMousePosition;
-        b.broadcast();
+        b.broadcast(this._lastMousePosition, e);
```

Comment:
I remember seeing up above that there was a getter.  Can we use the getter here instead?